packages/features/auth/lib/next-auth-options.ts

```diff
@@ -537,6 +537,33 @@ export const AUTH_OPTIONS: AuthOptions = {
           },
         });
 
+        /* --- START FIX LEGACY ISSUE WHERE 'identityProviderId' was accidentally set to userId --- */
```

Comment:
@emrysal Should we write a migration to fix this for all users in 1 go?

Reply:
Opted against it as I'm not sure we have the identityProviderId for all Google accounts, looks like the account system is newer - we should revisit in a ~2 months and drop this code.